lib/colvars/colvarmodule.h

```diff
@@ -83,6 +83,15 @@ class colvarmodule {
 
   /// Defining an abstract real number allows to switch precision
   typedef  double    real;
+
+  /// Override std::pow with a product for n positive integer
+  static inline real integer_power(real x, int n)
```

Comment:
check out https://github.com/lammps/lammps/blob/master/src/math_special.h#L67 for a more complete (and optmized) implementation of `pow(double,int)`